modules/ip/datapath/arp_output_request.c

```diff
@@ -26,53 +27,47 @@ enum {
 	EDGE_COUNT,
 };
 
-static struct rte_ring *nexthop_ring;
-
 int arp_output_request_solicit(struct nexthop *nh) {
 	int ret;
 	if (nh == NULL)
 		return errno_set(EINVAL);
 	ip4_nexthop_incref(nh);
-	ret = rte_ring_enqueue(nexthop_ring, nh);
+	ret = slow_path_post(RTE_ETHER_TYPE_IPV4, (void *)nh);
```

Comment:
I don't think `RTE_ETHER_TYPE_IPV4` is suitable here. You should define your own value `SLOW_PATH_ARP_SOLICIT`.

Reply:
fixed